src/Tizen.NUI/src/public/Animation.cs

```diff
@@ -493,6 +507,78 @@ public int[] EndTime
             }
         }
 
+        /// <summary>
+        /// Get the list of the properties of the animation.
+        /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<string> PropertyList
+        {
+            get
+            {
+                if (null == _propertyList)
+                {
+                    _propertyList = new List<string>();
+                }
+
+                return _propertyList;
+            }
+        }
+
+        /// <summary>
+        /// Get the list of the destination value for each property of the animation.
+        /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<string> DestValueList
```

Comment:
Framework Design Guidelines don't recommend using List<T> in public APIs.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections

❌ DO NOT use ArrayList or List in public APIs.
These types are data structures designed to be used in internal implementation, not in public APIs. List is optimized for performance and power at the cost of cleanness of the APIs and flexibility. For example, if you return List, you will not ever be able to receive notifications when client code modifies the collection. Also, List exposes many members, such as BinarySearch, that are not useful or applicable in many scenarios. The following two sections describe types (abstractions) designed specifically for use in public APIs.

I think you can use IList<T> instead of List<T>.